File: PRAK205-2310817120012-RirinCitraLestari.py
import math
A, B = map(float,input().split())
C = math.sqrt(A**2 + B**2)
alas = B
tinggi = A
keliling = A + B + C
luas = 0.5 * A * B
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm^2")

File: PRAK205-2310817120012-RirinCitraLestari.c
#include <stdio.h>
#include <math.h>
int main() {
    double A, B, C, alas, tinggi, keliling, luas;
    scanf("%lf", &A);
    scanf("%lf", &B);
    C = sqrt(A * A + B * B);
    alas = B;
    tinggi = A;
    keliling = A + B + C;
    luas = 0.5 * A * B;
    printf("Alas = %.2lf cm\n", alas);
    printf("Tinggi = %.2lf cm\n", tinggi);
    printf("Keliling = %.2lf cm\n", keliling);
    printf("Luas = %.2lf cm^2\n", luas);
    return 0;
}